<!-- -*- xml -*- -->
<launch>	
  <!-- Launch the tracking node -->
  <node pkg="visp_auto_tracker" type="visp_auto_tracker" name="visp_auto_tracker" output="screen">
    <param name="model_path" value="$(find visp_auto_tracker)/models" />
    <param name="model_name" value="pattern" />
    <param name="debug_display" value="True" />
	      
    <remap from="/visp_auto_tracker/camera_info" to="/usb_cam/camera_info"/>
    <remap from="/visp_auto_tracker/image_raw" to="/usb_cam/image_raw"/>
  </node>

  <!-- Launch the usb camera acquisition node -->
  <node pkg="usb_cam" type="usb_cam_node" name="usb_cam" output="screen">      
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="video_device" value="/dev/video1" />      
    <param name="pixel_format" value="yuyv" />
    <!-- rename the camera name topic into /camera/image_raw to match the one in visp_auto_tracker/models/calibration.ini file --> 
    <param name="camera_name" value="head_camera" />
    <!--<param name="camera_info_url" value="package://visp_auto_tracker/models/calibration.ini" type="string" />-->
    <param name="camera_info_url" value="file:///home/zhekai/.ros/camera_info/head_camera.yaml" type="string" />
  </node>

  <!-- Launch the custom qrcode_status node -->
  <node pkg="ur_moveit_myplan" type="cam_sta_pub.py" name="cam_sta_pub" output="screen">
  </node>

</launch>
